src/main/scala/chiseltest/legacy/backends/verilator/Utils.scala

```diff
@@ -419,7 +419,7 @@ private[chiseltest] object verilogToVerilator extends EditableBuildCSimulatorCom
     if (staleObjDri.exists()) {
       println(s"Deleting stale Verilator object directory: $staleObjDri")
       if (!firrtl.FileUtils.deleteDirectoryHierarchy(staleObjDri)) {
-        throw BackendException(s"Deleting $staleObjDri failed!")
+        throw new RuntimeException(s"Deleting $staleObjDri failed!")
```

Comment:
Why did you change the exception type here?

Reply:
If there is no good reason for doing so, then please revert back.

Reply:
`BackendException` adds the message "Unknown backend: xxx. Backend should be firrtl, verilator, ivl, vcs, or glsim" which is misleading/irrelevant to this situation (directory deletion failed). I had mistakenly used it in #303 and only discovered my mistake while testing this PR.

Reply:
OK. No need to change this then.